src/python/sortingview/gui/extensions/spikeamplitudes/spikeamplitudes.py

```diff
@@ -22,7 +22,7 @@ def _compute_peak_channel_index_from_average_waveform(average_waveform):
     modules=['sortingview']
 )
 @serialize_wrapper
-def fetch_spike_amplitudes(snippets_h5: str, unit_id: int):
+def fetch_spike_amplitudes(snippets_h5: str, unit_id: Union[int, List[int]]):
```

Comment:
It's important that calc tasks return spike trains for individual units, because then we don't separately cache [1, 2, 5] and [1, 2, 5, 6], if you know what I mean. The reason that we allow a list of unit ids is just so we can handle the case of merges. So I think it's okay the way it is.

Reply:
That's a great point about the caching--I hadn't thought of it. Given that, it might be good to drop a comment about it (I can imagine myself calling this with a group of IDs without thinking about the caching implication).

Reply:
I added some comments in the code.